applications/IGAStructuralMechanicsApplication/custom_elements/meshless_shell_element.cpp

```diff
@@ -583,45 +340,13 @@ void MeshlessShellElement::CalculateAndAddNonlinearKm(
 				for (unsigned int j = 0; j < check; j++)
 				{
 					K(3*n + i, 3*m + j) += (SD[0] * B11(3*n + i, 3*m + j) + SD[1] * B22(3 * n + i, 3 * m + j) + SD[2] * B12(3 * n + i, 3 * m + j))*weight;
```

Comment:
it may help readability to haave something like

unsigned int base_n = 3*n
unsigned int base_m = 3*m